pinot-core/src/test/java/org/apache/pinot/queries/ForwardIndexHandlerReloadQueriesTest.java

```diff
@@ -644,12 +737,15 @@ private void changePropertiesAndReloadSegment()
     invertedIndexEnabledColumns.add("column2");
     invertedIndexEnabledColumns.add("column7");
     indexLoadingConfig.setInvertedIndexColumns(invertedIndexEnabledColumns);
+    indexLoadingConfig.getInvertedIndexColumns().remove("column9");
     Set<String> noDictionaryColumns = new HashSet<>(_noDictionaryColumns);
     indexLoadingConfig.setNoDictionaryColumns(noDictionaryColumns);
     indexLoadingConfig.getNoDictionaryColumns().remove("column2");
     indexLoadingConfig.getNoDictionaryColumns().remove("column3");
     indexLoadingConfig.getNoDictionaryColumns().remove("column7");
     indexLoadingConfig.getNoDictionaryColumns().remove("column10");
+    indexLoadingConfig.getNoDictionaryColumns().add("column6");
+    indexLoadingConfig.getNoDictionaryColumns().add("column9");
```

Comment:
Do we handle basic validation (similar to what we did in noForwardIndex work) at the table config level ? 

For example, if someone wants to disable dictionary on a column, but does not remove the column from the `invertedIndexColumn` list in the tableConfig when updating the tableConfig, then we should throw error  right there. 

Other way would be to detect this invalid combination during reload but I guess that's too late.

Reply:
Yes. That's already there in TableConfigUtils. https://github.com/apache/pinot/blob/a22250270da250b03943624eae2a9466164b864c/pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java#L699